core/templates/pages/contributor-dashboard-page/translation-language-selector/translation-language-selector.component.spec.ts

```diff
@@ -258,4 +262,33 @@ describe('Translation language selector', () => {
         selectedLanguage);
     });
   });
+
+  it('should toggle dropdown', () => {
+    component.ngOnInit();
+
+    expect(component.dropdownShown).toBe(false);
+
+    component.toggleDropdown();
+
+    expect(component.dropdownShown).toBe(true);
+    expect(component.optionsFilter).toBe('');
+    expect(component.filteredOptions).toBe(component.options);
+
+    component.toggleDropdown();
+
+    expect(component.dropdownShown).toBe(false);
+  });
+
+  it('should filter language options based on the filter text', () => {
+    component.ngOnInit();
+
+    expect(component.filteredOptions.length).toBeGreaterThan(1);
```

Comment:
Please specify the elements of the list.

Reply:
Thanks for the suggestion! Added.